.gitignore

```diff
@@ -2,3 +2,7 @@
 /plotng.iml
 /plotng
 /bin/
+/plotng_linux_amd64.tar.gz
```

Comment:
pre-built binaries should not be checked into the repo

Reply:
yep, but sometime after package.sh for testing and a git add ., it may be included, just prevent

Reply:
can add to .gitignore to prevent checkin.